lib/cfa/augeas_parser.rb

```diff
@@ -193,6 +207,15 @@ def save_to_augeas(aug, prefix)
       end
     end
 
+    def ==(other)
+      other.class == self.class &&
+        other.data == data
```

Comment:
it can be done same way as above. I think here should be also added that data is order sensitive, so if two lines as switched, then it is not `==`